### ISSUE TITLE
IllegalArgumentException: Unexpected char 0x662 at 49 in Content-Disposition value: form-data; name="media[0]"; filename="S...

### ISSUE THREAD

**Issue by loremattei** [wordpress-mobile/WordPress-Android#13194]
**Sentry Url**: https://sentry.io/share/issue/6cf330eb8d4e4260ac3133f059edc772/
**User Count**: 67
**Count**: 710
**First Release**: org.wordpress.android@15.9-rc-1+930
**First Seen**: 2020-10-06T20:15:22.396000Z
**Last Seen**: 2020-10-21T10:42:12Z
**24 Hours**: 288
**30 Days**: 703

```
IllegalArgumentException: Unexpected char 0x662 at 49 in Content-Disposition value: form-data; name="media[0]"; filename="Screenshot_٢٠٢٠١٠٢١_١٣٣٧٤٥.jpg"
     at org.wordpress.android.fluxc.network.rest.wpcom.media.RestUploadRequestBody.buildMultipartBody(RestUploadRequestBody.java:78)
     at org.wordpress.android.fluxc.network.rest.wpcom.media.RestUploadRequestBody.<init>(RestUploadRequestBody.java:35)
     at org.wordpress.android.fluxc.network.rest.wpcom.media.MediaRestClient.uploadMedia(MediaRestClient.java:156)
     at org.wordpress.android.fluxc.store.MediaStore.performUploadMedia(MediaStore.java:768)
     at org.wordpress.android.fluxc.store.MediaStore.onAction(MediaStore.java:482)
...
(7 additional frame(s) were not displayed)
```

**Comment by develric** [comment]
I could reproduce this one in the following way:

- Download in your device/emu this [image](https://drive.google.com/file/d/1rJnc48ueXP4jk4h4oT0VKm1BKC8VBjnx/view?usp=sharing)
- Create a post and add an image block
- try to upload into the image block the downloaded image above
- boom

Verified that 15.8 does not have this issue but it's present in current 15.9 production so it seems probably good candidate for an hotfix if we can fix it. @planarvoid do you think you could have a look into this? Let me know, thanks 🙇

**Comment by planarvoid** [comment]
[This is related stackoverflow issue](https://stackoverflow.com/questions/58869988/unexpected-char-0x662-at-42-in-content-disposition-value-form-data-name-userf)

**Comment by develric** [comment]
closed by #13205
